### PR TITLE
Show full (un-ellipsed) release name when clicking on it.

### PR DESCRIPTION
**Pull request by p0psicles**
- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

image

### CODE REVIEW

**Comment by adaur** [comment]
Could this be done on hover?

**Comment by p0psicles** [comment]
Yes it could. But I'm not sure if you should want that?

**Comment by p0psicles** [comment]
Also we already have a tool tip. But that doesn't work that well. Especially not on mobile.

**Comment by adaur** [comment]
I don't know, I wouldn't try to click it but that's just me

File: static/js/home/snatch-selection.js [hunk 1]
```diff
@@ -238,4 +238,15 @@ MEDUSA.home.snatchSelection = function() {
             $('#wrapper').prop('data-history-toggle', 'show');
         });
     });
+
+    $('.release-name-ellipses, .release-name-ellipses-toggled').on('click', function(el) {
+        let target = $(el.currentTarget);
```
Comment (OmgImAlexis) [inline static/js/home/snatch-selection.js]: Switch this to const.

File: static/js/home/snatch-selection.js [hunk 2]
```diff
@@ -238,4 +238,15 @@ MEDUSA.home.snatchSelection = function() {
...
+        if (target.hasClass('release-name-ellipses')) {
```
Comment (OmgImAlexis) [inline static/js/home/snatch-selection.js]: Use [switchclass()](https://api.jqueryui.com/switchclass/) for this.

**Pull request merged**